### PR TITLE
Edit Participants and Trading Model

### PR DESCRIPTION
**Pull request by twzjwang**

### CODE REVIEW

**Comment by jserv** [comment]
Can you run `make indent` and then enforce consistent formation?

**Comment by twzjwang** [comment]
> Can you run `make indent` and then enforce consistent formation?

I get an error when running `make indent`.

```
latexindent -l paper.tex > paper-indent.tex
log4cxx: setFile(/Users/wang/Library/Application Support/MiKTeX/texmfs/data/miktex/log/latexindent.log,true) call failed.
log4cxx: IO Exception : status code = 13
log4cxx: No output stream or file set for the appender named [RollingLogFile].
Can't locate Log/Log4perl.pm in @INC (you may need to install the Log::Log4perl module) (@INC contains: /Library/Application Support/MiKTeX/texmfs/install/scripts/latexindent /Library/Perl/5.18/darwin-thread-multi-2level /Library/Perl/5.18 /Network/Library/Perl/5.18/darwin-thread-multi-2level /Network/Library/Perl/5.18 /Library/Perl/Updates/5.18.4 /System/Library/Perl/5.18/darwin-thread-multi-2level /System/Library/Perl/5.18 /System/Library/Perl/Extras/5.18/darwin-thread-multi-2level /System/Library/Perl/Extras/5.18 .) at /Library/Application Support/MiKTeX/texmfs/install/scripts/latexindent/LatexIndent/LogFile.pm line 22.
BEGIN failed--compilation aborted at /Library/Application Support/MiKTeX/texmfs/install/scripts/latexindent/LatexIndent/LogFile.pm line 22.
Compilation failed in require at /Library/Application Support/MiKTeX/texmfs/install/scripts/latexindent/LatexIndent/Document.pm line 25.
BEGIN failed--compilation aborted at /Library/Application Support/MiKTeX/texmfs/install/scripts/latexindent/LatexIndent/Document.pm line 25.
Compilation failed in require at /Library/Application Support/MiKTeX/texmfs/install/scripts/latexindent/latexindent.pl line 27.
BEGIN failed--compilation aborted at /Library/Application Support/MiKTeX/texmfs/install/scripts/latexindent/latexindent.pl line 27.
make: *** [indent] Error 2
```

I am trying to solve it.

**Comment by jserv** [comment]
`latexindent` is known to work with Ubuntu Linux 18.04 LTS.

File: paper.tex [hunk 1]
```diff
@@ -77,27 +77,27 @@ \section{\normalsize\textbf{Related Work}}
...
 
 \section{\normalsize\textbf{System Architecture}}
-Our proposed data marketplace framework is a 3-tier decentralized architecture with a registrar who is responsible for marketplace registration in order to put participants' information on distributed ledgers for validation, brokers who interact between data providers and consumers, including data uploading, product metadata generation and trading process, data providers who upload data, and data consumers who search for interested products and issue a new trade.
+Our proposed data marketplace framework is a 3-tier decentralized architecture with a registrar who is responsible for marketplace registration in order to put participants' information on distributed ledgers for validation, data providers who upload data, data consumers who search for interested products and issue a new trade, and brokers who interact between data providers and consumers, including data uploading, product metadata generation and trading process.
```
Comment (jserv) [inline paper.tex]: May we avoid using "upload"? Instead, "publish" might be better.

**Pull request merged**